static/compiler-service.ts

```diff
@@ -429,11 +431,9 @@ export class CompilerService {
         const stdout = result.stdout ?? [];
         // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
         const stderr = result.stderr ?? [];
-        // TODO: Make its own const variable at top of module?
```

Comment:
nice! (unrelated to this change?)

Reply:
Yes. Was doing a bunch of things at the same time and ran out of time to separate it into various PRs, sorry!

Reply:
n oworries! Just making sure things that aren't meant to get in aren't slipping before they're ready! thanks!